Dockerfile

```
@@ -0,0 +1,11 @@
+FROM registry.access.redhat.com/ubi8/nodejs-12:1-98.1634036321 as builder
```

Since this is not a multistep build you dont need to alias the builder image
```suggestion
FROM registry.access.redhat.com/ubi8/nodejs-12:1-98.1634036321
```